### ISSUE TITLE
sbt 1.0.+ support

### ISSUE THREAD

**Issue by l15k4** [tim-group/sbt-utc#3]
Hey, would you pls consider releasing version compatible with sbt 1.0.4 ?

```
sbt.librarymanagement.ResolveException: unresolved dependency: com.timgroup#sbt-utc;0.0.14: not found
```

**Comment by flicken** [comment]
Yes, happy to do it.  To set expectations, I may not have time to figure out how to do this until early (or even mid) January.   

I expect this will help me: http://www.scala-sbt.org/1.x/docs/Cross-Build-Plugins.html

Pull requests welcome to speed things along!

**Comment by jiminhsieh** [comment]
@flicken Can I work on this one? I will be happy to send the PR. :)

**Comment by flicken** [comment]
@jiminhsieh Yes, PRs welcome!

**Comment by flicken** [comment]
Re-opening until published.

**Comment by clintmcgill** [comment]
Is this ever going to be published? Would really like to have this for a SBT 1.1.x project.
